### ISSUE TITLE
LSQ+ V2 per_channel模式效果很差

### ISSUE THREAD

**Issue by frangkyy** [ZouJiu1/LSQplus#14]
您好，您有试过LSQ+weight的per_channel模式吗，这个量化出来的效果很差，不如per_tensor的，发现出来的weight的scale比per_tensor的大一个数量级，您有思路解决这一问题吗

**Comment by frangkyy** [comment]
发现了，第204行代码的EMA写错了，这一行和前一行代码应该改成：
self.s.data = self.s.data*0.9 + 0.1*torch.max(torch.stack([torch.abs(mean-3*std), torch.abs(mean + 3*std)]), dim=0)[0]/self.div

**Comment by lpz7** [comment]
![屏幕截图 2024-02-27 155838](https://github.com/ZouJiu1/LSQplus/assets/73594726/a723b3d1-038d-4584-8819-91be06739533)
当我使用per_channel量化yolov5时，会报这个错误，您知道这个问题怎么解决吗
